config/core/deployments/pingsource-jobrunner.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
```

Comment:
Values here is taken from the programmatic deployment creation here: https://github.com/knative/eventing/blob/f8b42c3b6e13097bc53283076a686f4fdb50ab52/pkg/reconciler/pingsource/controller/resources/jobrunner.go#L43